stingray/modeling/parameterestimation.py

```diff
@@ -297,34 +290,73 @@ def fit(self, lpost, t0, neg=True, scipy_optimize_options=None):
             params = [getattr(newmod,name) for name in newmod.param_names]
             bounds = [p.bounds for p in params if not np.any([p.tied, p.fixed])]
 
+            if len(bounds) > 0 and self.fitmethod not in ["L-BFGS-B",
+                                                          "TNC",
+                                                          "SLSQP"]:
+                logging.warning("Fitting method %s "%self.fitmethod +
+                                "cannot incorporate the bounds you set!")
+
+            if len(bounds) == 0 or self.fitmethod not in ["L-BFGS-B",
```

Comment:
This might be limiting if new methods are implemented in Scipy that accept bounds. 
Why forcing `use_bounds` to False? If they choose a method that doesn't accept bounds, they might be given a warning and if they persevere... well, _perseverare diabolicum_